### ISSUE TITLE
[v2.0.0-alpha.71] Did not install docusaurus-plugin-sass successfully

### ISSUE THREAD

**Issue by MisterFISHUP** [facebook/docusaurus#4397]
## 🐛 Bug Report

I tried to install `docusaurus-plugin-sass` but got a lot of errors in terminal.

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes.

## To Reproduce

I just ran the following commands to init a project and to install `docusaurus-plugin-sass`:

1. `npx @docusaurus/init@latest init my-website classic`
2. `cd my-website`
3. `npm install --save docusaurus-plugin-sass`

## Expected behavior

Install `docusaurus-plugin-sass` successfully without errors.

## Actual Behavior

I got these messages in terminal right after running the three commands above.

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: my-website@0.0.0
npm ERR! Found: @docusaurus/core@2.0.0-alpha.71
npm ERR! node_modules/@docusaurus/core
npm ERR!   @docusaurus/core@"2.0.0-alpha.71" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @docusaurus/core@"^2.0.0" from docusaurus-plugin-sass@0.1.12
npm ERR! node_modules/docusaurus-plugin-sass
npm ERR!   docusaurus-plugin-sass@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/fishup/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/fishup/.npm/_logs/2021-03-11T11_43_46_414Z-debug.log
```

## Your Environment

- Docusaurus version used: v2.0.0-alpha.71
- Environment name and version: node v14.16.0
- Operating system and version (desktop or mobile): ubuntu 20.04

**Comment by slorber** [comment]
Sorry but I can't reproduce using a quite similar env.

Assuming it is a temporary issue.

Please try again, try also on another computer, and if it still fails, send a full log + a repro

My full log:

```
 ~/Desktop  npx @docusaurus/init@latest init my-website classic                                                                                                                       ✓  10007  19:25:58

npx: installed 39 in 6.603s

Creating new Docusaurus project ...

Installing dependencies with: yarn
yarn install v1.22.5
info No lockfile found.
[1/4] Resolving packages...
warning @docusaurus/core > webpack-dev-server > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
warning @docusaurus/core > webpack-dev-server > chokidar > fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
warning @docusaurus/core > @docusaurus/utils > intl-locales-supported@1.8.12: bad publish
warning @docusaurus/core > webpack > watchpack > watchpack-chokidar2 > chokidar@2.1.8: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
warning @docusaurus/core > webpack-dev-server > chokidar > braces > snapdragon > source-map-resolve > resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
warning @docusaurus/core > webpack-dev-server > chokidar > braces > snapdragon > source-map-resolve > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
[2/4] Fetching packages...
[3/4] Linking dependencies...
warning "@docusaurus/preset-classic > @docusaurus/theme-classic > react-toggle@4.1.1" has incorrect peer dependency "react@^15.3.0 || ^16.0.0".
warning "@docusaurus/preset-classic > @docusaurus/theme-classic > react-toggle@4.1.1" has incorrect peer dependency "react-dom@^15.3.0 || ^16.0.0".
warning "@docusaurus/preset-classic > @docusaurus/theme-search-algolia > @docsearch/react@3.0.0-alpha.34" has unmet peer dependency "@types/react@>= 16.8.0 < 18.0.0".
warning "@docusaurus/preset-classic > @docusaurus/theme-search-algolia > @docsearch/react > @algolia/autocomplete-preset-algolia@1.0.0-alpha.44" has unmet peer dependency "@algolia/client-search@^4.5.1".
[4/4] Building fresh packages...
success Saved lockfile.
Done in 31.85s.


Success! Created my-website
Inside that directory, you can run several commands:

  yarn start
    Starts the development server.

  yarn build
    Bundles the app into static files for production.

  yarn deploy
    Publish website to GitHub pages.

We suggest that you begin by typing:

  cd my-website
  yarn start

Happy hacking!
 ~/Desktop  cd my-website                                                                                                                                                             ✓  10008  19:26:43
 ~/Desktop/my-website  npm install --save docusaurus-plugin-sass                                                                                                                      ✓  10009  19:27:01
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated intl-locales-supported@1.8.12: bad publish
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN rm not removing /Users/sebastienlorber/Desktop/my-website/node_modules/.bin/semver as it wasn't installed by /Users/sebastienlorber/Desktop/my-website/node_modules/semver
npm WARN rm not removing /Users/sebastienlorber/Desktop/my-website/node_modules/.bin/rimraf as it wasn't installed by /Users/sebastienlorber/Desktop/my-website/node_modules/rimraf
npm WARN rm not removing /Users/sebastienlorber/Desktop/my-website/node_modules/.bin/autoprefixer as it wasn't installed by /Users/sebastienlorber/Desktop/my-website/node_modules/autoprefixer
npm WARN rm not removing /Users/sebastienlorber/Desktop/my-website/node_modules/.bin/mkdirp as it wasn't installed by /Users/sebastienlorber/Desktop/my-website/node_modules/mkdirp
npm WARN rm not removing /Users/sebastienlorber/Desktop/my-website/node_modules/.bin/json5 as it wasn't installed by /Users/sebastienlorber/Desktop/my-website/node_modules/json5
npm WARN rm not removing /Users/sebastienlorber/Desktop/my-website/node_modules/.bin/detect as it wasn't installed by /Users/sebastienlorber/Desktop/my-website/node_modules/detect-port
npm WARN rm not removing /Users/sebastienlorber/Desktop/my-website/node_modules/.bin/detect-port as it wasn't installed by /Users/sebastienlorber/Desktop/my-website/node_modules/detect-port
npm WARN rm not removing /Users/sebastienlorber/Desktop/my-website/node_modules/.bin/cssesc as it wasn't installed by /Users/sebastienlorber/Desktop/my-website/node_modules/cssesc

> fsevents@1.2.13 install /Users/sebastienlorber/Desktop/my-website/node_modules/watchpack-chokidar2/node_modules/fsevents
> node install.js

  SOLINK_MODULE(target) Release/.node
  CXX(target) Release/obj.target/fse/fsevents.o
  SOLINK_MODULE(target) Release/fse.node

> fsevents@1.2.13 install /Users/sebastienlorber/Desktop/my-website/node_modules/webpack-dev-server/node_modules/fsevents
> node install.js

  SOLINK_MODULE(target) Release/.node
  CXX(target) Release/obj.target/fse/fsevents.o
  SOLINK_MODULE(target) Release/fse.node

> node-sass@4.14.1 install /Users/sebastienlorber/Desktop/my-website/node_modules/node-sass
> node scripts/install.js

Cached binary found at /Users/sebastienlorber/.npm/node-sass/4.14.1/darwin-x64-83_binding.node

> core-js@3.9.1 postinstall /Users/sebastienlorber/Desktop/my-website/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon:
> https://opencollective.com/core-js
> https://www.patreon.com/zloirock

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)


> core-js-pure@3.9.1 postinstall /Users/sebastienlorber/Desktop/my-website/node_modules/core-js-pure
> node -e "try{require('./postinstall')}catch(e){}"


> node-sass@4.14.1 postinstall /Users/sebastienlorber/Desktop/my-website/node_modules/node-sass
> node scripts/build.js

Binary found at /Users/sebastienlorber/Desktop/my-website/node_modules/node-sass/vendor/darwin-x64-83/binding.node
Testing binary
Binary is fine
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN docusaurus-plugin-sass@0.1.12 requires a peer of @docusaurus/core@^2.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN react-toggle@4.1.1 requires a peer of react@^15.3.0 || ^16.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN react-toggle@4.1.1 requires a peer of react-dom@^15.3.0 || ^16.0.0 but none is installed. You must install peer dependencies yourself.

+ docusaurus-plugin-sass@0.1.12
added 716 packages from 170 contributors, removed 135 packages, updated 1362 packages and audited 2081 packages in 634.318s

180 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities



   ╭───────────────────────────────────────────────────────────────╮
   │                                                               │
   │      New major version of npm available! 6.14.8 → 7.6.2       │
   │   Changelog: https://github.com/npm/cli/releases/tag/v7.6.2   │
   │               Run npm install -g npm to update!               │
   │                                                               │
   ╰───────────────────────────────────────────────────────────────╯

 ~/Desktop/my-website 
```

**Comment by MisterFISHUP** [comment]
Thank you for your reply. `npm install --save docusaurus-plugin-sass` still didn't work for me, but then I checked the plugin's [github repo](https://github.com/rlamana/docusaurus-plugin-sass) and tried with `yarn add docusaurus-plugin-sass` and it worked worked like a charm.
